### PR TITLE
Fix 1.8.0 upgrade docs for new CRDs

### LINKED ISSUE #4951: helm upgrade from 1.7.10 to 1.8.0 fails on gateway pod
**Describe the bug**
Attempting to use helm to upgrade server from 1.7.10 to 1.8.0, the 1.8.0 gateway pod goes into CrashLoopBackOff.

**To Reproduce**
1. Start with Gloo Edge OSS 1.7.10

```
% glooctl version
Client: {"version":"1.7.10"}
Server: {"type":"Gateway","kubernetes":{"containers":[{"Tag":"1.7.10","Name":"discovery","Registry":"quay.io/solo-io"},{"Tag":"1.7.10","Name":"gateway","Registry":"quay.io/solo-io"},{"Tag":"1.7.10","Name":"gloo-envoy-wrapper","Registry":"quay.io/solo-io"},{"Tag":"1.7.10","Name":"gloo","Registry":"quay.io/solo-io"}],"namespace":"gloo-system"}}
```

2. Observe that `glooctl version` runs as expected.

```
% glooctl version
----------
glooctl binary version (1.8.0) differs from server components (v1.7.10) by at least a minor version.
Consider running:
glooctl upgrade --release=v1.7.10
----------

Client: {"version":"1.8.0"}
Server: {"type":"Gateway","kubernetes":{"containers":[{"Tag":"1.7.10","Name":"discovery","Registry":"quay.io/solo-io"},{"Tag":"1.7.10","Name":"gateway","Registry":"quay.io/solo-io"},{"Tag":"1.7.10","Name":"gloo-envoy-wrapper","Registry":"quay.io/solo-io"},{"Tag":"1.7.10","Name":"gloo","Registry":"quay.io/solo-io"}],"namespace":"gloo-system"}}
```

3. Attempt to upgrade server to 1.8.0 using helm.

```
helm upgrade -n gloo-system gloo gloo/gloo --version=v1.8.0
Release "gloo" has been upgraded. Happy Helming!
NAME: gloo
LAST DEPLOYED: Tue Jun 29 17:13:34 2021
NAMESPACE: gloo-system
STATUS: deployed
REVISION: 2
TEST SUITE: None
```

3. Gateway pod fails to start and enters CrashLoopBackOff state. See error in gateway pod logs.

```
% kubectl logs -n gloo-system pod/gateway-7767b977df-vfxkx
{"level":"info","ts":1625001404.329821,"logger":"gateway.v1.event_loop","caller":"v1/setup_event_loop.sk.go:57","msg":"event loop started","version":"1.8.0"}
{"level":"fatal","ts":1625001404.4591255,"logger":"gateway","caller":"setuputils/main_setup.go:88","msg":"error in setup: creating base VirtualHostOption resource client: list check failed: the server could not find the requested resource (get virtualhostoptions.gateway.solo.io)","version":"1.8.0","stacktrace":"github.com/solo-io/gloo/pkg/utils/setuputils.Main\n\t/workspace/gloo/pkg/utils/setuputils/main_setup.go:88\ngithub.com/solo-io/gloo/projects/gateway/pkg/setup.Main\n\t/workspace/gloo/projects/gateway/pkg/setup/setup.go:13\nmain.main\n\t/workspace/gloo/projects/gateway/cmd/main.go:11\nruntime.main\n\t/usr/local/go/src/runtime/proc.go:225"}
```

**Expected behavior**
Upgrade works with no problems

**Additional context**
- Gloo Edge version 1.7.10 to 1.8.0
- Kubernetes version 1.18.2 (kind cluster)

### PR DESCRIPTION
**Pull request by jameshbarton**
# Description

This PR fixes a couple of holes in the docs specific to 1.8.0 upgrades.

Closes #4951

### CODE REVIEW

File: docs/content/operations/upgrading/1.8.0.md [hunk 1]
```diff
@@ -28,10 +28,24 @@ Helm maintainers, given the risk associated with changing CRDs. Given this limit
...
+helm repo update
+helm upgrade -n gloo-system gloo gloo/gloo --version=v1.8.0
+```
+{{% /tab %}}
+{{% tab name="Gloo Edge Enterprise - Helm 3" %}}
 ```shell script
 kubectl apply -f https://raw.githubusercontent.com/solo-io/gloo/v1.8.0/install/helm/gloo/crds/gateway.solo.io_v1_VirtualHostOption.yaml
 kubectl apply -f https://raw.githubusercontent.com/solo-io/gloo/v1.8.0/install/helm/gloo/crds/gateway.solo.io_v1_RouteOption.yaml
+helm repo update
+helm upgrade -n gloo-system gloo gloo/gloo --version=v1.8.0
```
Comment (sam-heilbron) [inline docs/content/operations/upgrading/1.8.0.md:45]: `glooe/gloo-ee` ?

Comment (jameshbarton) [reply docs/content/operations/upgrading/1.8.0.md]: Nice catch - made that change

**Review by sam-heilbron — APPROVED** [review]

File: docs/content/operations/upgrading/1.8.0.md [hunk 2]
```diff
@@ -28,10 +28,24 @@ Helm maintainers, given the risk associated with changing CRDs. Given this limit
 You can add the new CRDs to your cluster in two ways. The first is to supply a URL that points to the CRD template in the public
 [Gloo Edge GitHub repository](https://github.com/solo-io/gloo)
 
+{{< tabs >}}
+{{% tab name="Gloo Edge - Helm 3" %}}
+```shell script
+kubectl apply -f https://raw.githubusercontent.com/solo-io/gloo/v1.8.0/install/helm/gloo/crds/gateway.solo.io_v1_VirtualHostOption.yaml
+kubectl apply -f https://raw.githubusercontent.com/solo-io/gloo/v1.8.0/install/helm/gloo/crds/gateway.solo.io_v1_RouteOption.yaml
+helm repo update
+helm upgrade -n gloo-system gloo gloo/gloo --version=v1.8.0
```
Comment (jenshu) [inline docs/content/operations/upgrading/1.8.0.md:37]:
```suggestion
helm upgrade -n gloo-system gloo gloo/gloo --version=1.8.0
```

File: docs/content/operations/upgrading/1.8.0.md [hunk 3]
```diff
@@ -28,10 +28,24 @@ Helm maintainers, given the risk associated with changing CRDs. Given this limit
...
+```
+{{% /tab %}}
+{{% tab name="Gloo Edge Enterprise - Helm 3" %}}
 ```shell script
 kubectl apply -f https://raw.githubusercontent.com/solo-io/gloo/v1.8.0/install/helm/gloo/crds/gateway.solo.io_v1_VirtualHostOption.yaml
 kubectl apply -f https://raw.githubusercontent.com/solo-io/gloo/v1.8.0/install/helm/gloo/crds/gateway.solo.io_v1_RouteOption.yaml
+helm repo update
+helm upgrade -n gloo-system glooe gloo/gloo-ee --version=v1.8.0
```
Comment (jenshu) [inline docs/content/operations/upgrading/1.8.0.md:45]:
```suggestion
helm upgrade -n gloo-system glooe gloo/gloo-ee --version=1.8.0
```

File: docs/content/operations/upgrading/1.8.0.md (hunk 2 above)
Comment (jenshu) [reply docs/content/operations/upgrading/1.8.0.md:37]: not sure if it works with the `v`?

Comment (jenshu) [reply docs/content/operations/upgrading/1.8.0.md:37]: oh looks like it does work with v as well.  although elsewhere in our docs we don't use it

**Review by jenshu — APPROVED** [review]

**Pull request merged**